bliss/encoder/variational_dist.py

```diff
@@ -153,10 +154,10 @@ def compute_nll(self, true_tile_cat: TileCatalog):
             loss += gal_flux_loss
 
         # galaxy properties loss
-        galsim_true_vals = rearrange(true_tile_cat["galaxy_params"], "b ht wt 1 d -> b ht wt d")
-        for i, param_name in enumerate(self.GALSIM_NAMES):
-            galsim_pn = f"galsim_{param_name}"
-            loss_term = -q[galsim_pn].log_prob(galsim_true_vals[..., i] + 1e-9) * true_gal_bools
-            loss += loss_term
+        # galsim_true_vals = rearrange(true_tile_cat["galaxy_params"], "b ht wt 1 d -> b ht wt d")
+        # for i, param_name in enumerate(self.GALSIM_NAMES):
+        #     galsim_pn = f"galsim_{param_name}"
+        #     loss_term = -q[galsim_pn].log_prob(galsim_true_vals[..., i] + 1e-9) * true_gal_bools
+        #     loss += loss_term
```

Comment:
Our flake8 checks don't allow commented out code. If it's problematic to estimate these galaxy parameters for our DC2 benchmarking, we could pass a binary flag to the constructor of this class (set in our hydra configs) to control whether galaxy params are estimated

Reply:
add flag `include_galaxy_property_loss`